extension/src/popup/helpers/formatters.ts

```diff
@@ -44,7 +45,7 @@ export const formatAmount = (
   if (cleaned.indexOf(".") !== -1) {
     const parts = cleaned.split(".");
     parts[0] = decimal.format(Number(parts[0].slice(0, maxDigits))).toString();
-    parts[1] = parts[1].slice(0, 7);
+    parts[1] = parts[1].slice(0, decimals);
```

Comment:
you could import the default decimals value (7) here: `decimals || classicDefaultPrecision` (not sold on that var name)